src/pages/App/index.tsx

```diff
@@ -1,38 +1,80 @@
 /* eslint-disable react/no-unstable-nested-components */
-/* eslint-disable @typescript-eslint/no-use-before-define */
+import { animated, config, useSpring, useTrail } from '@react-spring/web';
 import i18n from 'i18next';
-import React, { Suspense, useEffect } from 'react';
+import React, { Suspense, useEffect, useState } from 'react';
 import { I18nextProvider, useTranslation } from 'react-i18next';
-import styled from 'styled-components';
 
 // components
-import AppIcon from '../../components/AppIcon';
 import Alert from '../../components/Text/Alert';
 
-// theme
-import { animation } from '../../theme';
-
 // apps
 import { loadApp } from '../../apps';
+import { AppManifest } from '../../types';
+
+type AnimatedAppTitleProps = {
+  text: string;
+};
+
+const AnimatedAppTitle: React.FC<AnimatedAppTitleProps> = ({ text }) => {
+  const [isDisplaying, setIsDisplaying] = useState(true);
+
+  const letters = text.split('');
+
+  const trail = useTrail(letters.length, {
```

Comment:
can we alter the duration here to 2 seconds, with the build-in and build-out to 250ms? Just to further refine the animation - the app name screen is too long at 4 seconds